src/Tribe/Commerce/PayPal/Gateway.php

```diff
@@ -175,7 +183,14 @@ public function add_to_cart() {
 
 		foreach ( $product_ids as $ticket_id ) {
 			$ticket   = tribe( 'tickets.commerce.paypal' )->get_ticket( $post->ID, $ticket_id );
-			$quantity = absint( $_POST[ "quantity_{$ticket_id}" ] );
+
+			$quantity = 0;
+
+			if ( isset( $_POST['tribe_tickets'][ $ticket_id ]['quantity'] ) ) {
+				$quantity = absint( $_POST['tribe_tickets'][ $ticket_id ]['quantity'] );
+			} elseif ( isset( $_POST["quantity_{$ticket_id}"] ) ) {
+				$quantity = absint( $_POST["quantity_{$ticket_id}"] );
```

Comment:
* :no_entry_sign: Processing form data without nonce verification.
* :no_entry_sign: Array keys must be surrounded by spaces unless they contain a string or an integer.
* :no_entry_sign: There must be a single space after the first square bracket
* :no_entry_sign: There must be a single space before the last square bracket

<sub>hash: 15a2e88450b95c278be7e642fc10b34d</sub>